storage/innobase/xtrabackup/src/ds_compress.c

```diff
@@ -190,6 +190,7 @@ compress_write(ds_file_t *file, const void *buf, size_t len)
 	uint			i;
 	const char		*ptr;
 	ds_file_t		*dest_file;
+	my_bool simulate_compress_error = FALSE;
```

Comment:
doesn't this cause relase build failure? this variable used only on debug builds?

Reply:
@satya-bodapati this is always false on relwithdebinfo. Unless you are in debug with the debug flag set, then this will be changed to true.

Reply:
on upstream, when you have variables that are used only in DEBUG_EXECUTE_IF() cause release build failures.The variable is declared under  #ifndef DBUG_OFF. Maybe this is OK in PXB

Reply:
the way it's built here it will be used on relwithdebinfo, but it will always be false. Inside DEBUG_EXECUTE_IF we change it to true.

Reply:
ignore, it is used at on more place apart from DBUG_EXECUTE_IF()